packages/js/.eslintrc.js

```diff
@@ -1,6 +1,6 @@
 module.exports = {
   extends: ['../../.eslintrc.js'],
-  plugins: ['local-rules'],
+  plugins: ['import', 'promise', '@typescript-eslint', 'prettier', 'local-rules'],
```

Comment:
fixed the eslint + prettier configuration for the package, had to move local-rules file also to root